### PR TITLE
updated timer, fixed bug with words continuing to appear after game end

### PR DESCRIPTION
**Pull request by kel29**

### CODE REVIEW

File: frontend/src/wordInteractions.js [hunk 1]
```diff
@@ -84,21 +87,34 @@ function loadEntryForm () {
...
-    seconds.textContent = '0' + time;
-  } else if (time < 60) {
-    seconds.textContent = time;
+  const minutes = document.getElementById('minutes');
+  const hours = document.getElementById('hours');
+
+  const startTime = new Date(game.run.created_at);
+  const currentTime = new Date();
+
+  const secs = Math.round((currentTime - startTime) / 1000) % 60;
```
Comment (renburnett) [inline frontend/src/wordInteractions.js]: modulus woo!!!!

File: frontend/src/wordInteractions.js [hunk 2]
```diff
@@ -84,21 +87,34 @@ function loadEntryForm () {
...
+  secs < 10 ? seconds.textContent = 0 + secs : seconds.textContent = secs;
+  const mins = Math.round((currentTime - startTime) / 60000) % 60;
+  mins < 10 ? minutes.textContent = 0 + mins : minutes.textContent = mins;
+  const hrs = Math.round((currentTime - startTime) / 360000) % 24;
+  hrs < 10 ? hours.textContent = 0 + hrs : hours.textContent = hrs;
```
Comment (justinamaple) [inline frontend/src/wordInteractions.js]: I would probably pull this logic out into it's own function. Smart way to handle the leading 0's on the timer though :)

File: frontend/src/wordInteractions.js [hunk 3]
```diff
@@ -129,6 +145,8 @@ function gameOver () {
 
   document.getElementById('word-submission-div').classList.add('hidden');
   document.getElementById('words-to-type').innerHTML = '';
+  resetTimer();
+  clearInterval(WORD_POPULATION_ID);
```
Comment (justinamaple) [inline frontend/src/wordInteractions.js]: oh nice, figured there was something like this

File: frontend/src/wordInteractions.js (hunk 2 above)
Comment (kel29) [reply frontend/src/wordInteractions.js]: good point, will do

**Pull request merged**